initial-configuration/jenkins/jenkins-docker/config.xml

```diff
@@ -176,7 +176,7 @@
           <string>project_specific_override_repo</string>
           <string>__PROJECT_SPECIFIC_OVERRIDE_REPO__</string>
           <string>release_control_branch</string>
-          <string>feature/redhat</string>
+          <string>*/master</string>
```

Comment:
```suggestion
          <string>*/main</string>
```

Reply:
GIC institution node are the only ones we're supporting for redhat, and those repos have main as their base branch. (At some point main will be the base for all repos, but we're still migrating base pic-sure.)